exercises/bracket-push/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include "bracket_push.h"
+#include <stdio.h>
+
+static bool is_bbp(char c)
+{
+   return (c == '{') || (c == '}') ||
+       (c == '[') || (c == ']') || (c == '(') || (c == ')');
+}
+
+static bool matching(char a, char b)
+{
+   if (a == '}')
+      return b == '{';
+   if (a == ']')
+      return b == '[';
+   if (a == ')')
+      return b == '(';
+
+   return false;
+}
+
+bool bbp_okay(const char *input)
```

Comment:
Unusual acronyms in public interfaces are often frowned upon - maybe go with `is_paired()` or something a little more verbose?